pkgs/development/lua-modules/toml-edit/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
```

Comment:
I would prefer if the overrides could be in pkgs/development/lua-modules/overrides.nix. This way the src keeps getting updated in generated.nix. Was there any difficulty doing so ?

Reply:
Yup, that works and is much cleaner. Thanks for the tip!